app/views/provider_interface/reports/withdrawal_reports/show.html.erb

```diff
@@ -0,0 +1,40 @@
+<%= content_for :title, t('page_titles.provider.withdrawal_report') %>
+<%= content_for :before_content, breadcrumbs(t('page_titles.provider.reports') => provider_interface_reports_path,
+                                             t('page_titles.provider.withdrawal_report') => nil) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-l">
+      <span class="govuk-caption-l"> <%= @provider.name %></span>
+      <%= t('page_titles.provider.withdrawal_report') %>
+    </h1>
+  </div>
+</div>
+
+<% if @submitted_withdrawal_reason_count < 10 %>
+  <p class="govuk-body">
+    You’ll be able to see this report once it contains data from at least 10 candidates. This is to protect the privacy of candidates.
+  </p>
+<% else %>
+  <p class="govuk-body">Candidates who withdraw their application are asked to optionally select their reason or reasons.</p>
+  <p class="govuk-body"> They are asked this when:</p>
+  <ul class="govuk-list govuk-list--bullet">
+    <li>withdrawing before a decision on their application is made</li>
+    <li>withdraing after accepting an offer</li>
+  </ul>
+  <p class="govuk-body">
+    Candidates who have received an offer are not able to withdraw – they have to either accept or decline instead.
+  </p>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+      <h2 class="govuk-heading-m">Applications withdrawn in the <%= RecruitmentCycle.cycle_name %> recruitment cycle</h3>
```

Comment:
Can we add two lines of text here to give some context about the table. It should read:

<p class="govuk-body">Candidates can select multiple reasons, so the numbers in each column may not match the ‘Total’ number.</p>

<p class="govuk-body">This data has only been collected from 11 April 2023.</p>